lib/workers/repository/finalise/prune.ts

```diff
@@ -51,8 +51,8 @@ async function cleanUpBranches(
             'Autoclosing PR'
           );
           let newPrTitle = pr.title;
-          if (!pr.title.endsWith('- autoclosed')) {
-            newPrTitle += '- autoclosed';
+          if (!pr.title.endsWith(' - autoclosed')) {
```

Comment:
```suggestion
          if (!pr.title.endsWith('- autoclosed')) {
```
backward compabillity

Reply:
Good point, but in that case we should have both checks, no?

So that would be this:
```suggestion
          if (!pr.title.endsWith(' - autoclosed') && !pr.title.endsWith('- autoclosed')) {
```

Reply:
no, as my suggestion also matches the new pr title schema